typescript/.changeset/kind-terms-spend.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-onchain-agent": patch
```

Comment:
should this be minor? at least for consistency with the same change in python

Reply:
Ah, yeah we can do that